src/library-entry.js

```diff
@@ -15,6 +15,9 @@ export { default as gard } from "./jsx/gard.jsx";
 export { default as bgm } from "./jsx/bgm.jsx";
 export { default as fade } from "./jsx/fade.jsx";
 
+// renderHyPhyVision is for use in Galaxy
+export { defualt as renderHyPhyVision } from "./renderHyPhyVision.js";
```

Comment:
Typo in `default` here... I blame VS Code

Reply:
If only VS Code was to blame... I blame me. 😢